src/site/content/en/blog/badging-api/index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Badging for app icons
+subhead: The Badging API allows installed web apps to set an application-wide badge on the app icon.
+authors:
+  - petelepage
+description: The Badging API allows installed web apps to set an application-wide badge, shown in an operating-system-specific place associated with the application, such as the shelf or home screen. Badging makes it easy to subtly notify the user that there is some new activity that might require their attention, or it can be used to indicate a small amount of information, such as an unread count.
+date: 2018-12-11
+updated: 2019-10-08
+tags:
+  - post
+  - capabilities
+  - badging
+  - progressive-web-apps
+  - notifications
+  - origintrials
+  - fugu
+hero: hero.jpg
+alt: Phone showing several notification badges
+draft: true
+---
+
+{% Aside 'caution' %}
+  This API is part of the new
+  [capabilities project](https://developers.google.com/web/updates/capabilities),
+  and starting in Chrome 73 it is available as an [**origin trial**](#ot).
+  This post will be updated as the Badging API evolves.
+{% endAside %}
+
+## What is the Badging API? {: #what }
+
+<figure class="w-figure w-figure--inline-right">
+  <img  src="badges-on-windows.jpg" class="w-screenshot"
+        alt="Example of Twitter with eight notifications and another app showing a flag type badge.">
+  <figcaption>
+    Example of Twitter with eight notifications and another app showing a flag
+    type badge.
+  </figcaption>
+</figure>
+
+The Badging API allows installed web apps to set an application-wide badge,
+shown in an operating-system-specific place associated with the application
+(such as the shelf or home screen).
+
+Badging makes it easy to subtly notify the user that there is some new activity
+that might require their attention, or to indicate a small amount of
+information, such as an unread count.
+
+Badges tend to be more user-friendly than notifications, and can be updated
+with a much higher frequency, since they don't interrupt the user. And,
+because they don't interrupt the user, they don't need the user's permission.
+
+### Possible use cases {: #use-cases }
+
+Examples of sites that may use this API include:
+
+* Chat, email, and social apps, to signal that new messages have arrived, or to
+  show the number of unread items.
+* Productivity apps, to signal that a long-running background task (such as
+  rendering an image or video) has completed.
+* Games, to signal that a player action is required (e.g., in Chess, when it
+  is the player's turn).
+
+## Current status {: #status }
+
+| Step                                       | Status                       |
```

Comment:
I think you'll need to wrap this table in `<div class="w-table-wrapper">` for it to overflow and horizontally scroll on mobile. Otherwise it breaks the right edge.

I checked and devsite also has a `devsite-table-wrapper` class but perhaps they inject it as part of their custom markdown parse so it's transparent to the author. I think you're the first person to use a markdown table on the site so we didn't catch this until now :\

If you do the following it will work:

```
<div class="w-table-wrapper">

| Step                                       | Status                       |
| ------------------------------------------ | ---------------------------- |
| 1. Create explainer                        | [Complete][explainer]        |
| 2. Create initial draft of specification   | [Complete][spec]             |
| 3. Gather feedback & iterate on design     | [In progress](#feedback)     |
| **4. Origin trial**                        | [**In progress**](#ot)       |
| 5. Launch                                  | Not started                  |

</div>
```

The whitespace between the divs is important. Kinda gross but it does the job.

Reply:
cc @kaycebasques

Reply:
Added